pycon/templates/finaid/receipt_form.html

```diff
@@ -0,0 +1,65 @@
+{% extends "site_base.html" %}
```

Comment:
We would probably name this template something similar to the view name, e.g. `receipt_add` or `receipt_upload`. (noun_verb.html seems to be the pattern).